src/components/content/Dashboard/Dashboard.js

```diff
@@ -107,27 +104,23 @@ class Dashboard extends Component {
 				<Header userid={userid}/>
 				<div className="dashboard-container">
 					<div className="group-list-container">
-						<label className="dash-section-title">Groups</label>
-						<ul className="groups-list">
-							{displayGroups}
-							<li className="create-project-thumb">
-								<div className="create-project-thumb-body" onClick={() => this.handleCreateButton('group')}>
-									<img src={addIcon} alt="add icon"/>
-								</div>
-								<div className="create-project-thumb-footer">
-									<label> Create a Group </label>
-								</div>
-							</li>
-						</ul>
+							<label className="dash-section-title">Group Projects</label>
```

Comment:
Originally this was just displaying links to group dashboards, rather than specific projects created by groups. All of the projects a user is associated with--group AND personal-- are rendered in `displayProjects`.

Reply:
I fixed that, nice catch. Once I fixed the layout I forgot to switch that.